.github/workflows/build-macos.yml

```diff
@@ -75,36 +75,37 @@ jobs:
       fail-fast: false  # if a particular matrix build fails, don't skip the rest
       matrix:
         # Split job into 5 matrix builds, because GH actions provides 5 concurrent
-        # builds on macOS
+        # builds on macOS. This needs to be manually kept updated so that each
+        # of these builds take roughly the same time
         include:
           - { 
-            name: "x86_64 (cp36-cp310)",
+            name: "x86_64 (CPython 3.6 3.10 and above)",
```

Comment:
Isn't this CPython 3.6 and above?

Right now it sounds like: 3.6, 3.10, and 3.11+

Reply:
Ah geez this is 3.6 or 3.[double digit number].

That's hard to parse for me.

Reply:
It is what it sounds